data/src/main/java/com/linkedin/data/codec/ProtobufDataCodec.java

```diff
@@ -78,9 +81,17 @@ public ProtobufDataCodec(SymbolTable symbolTable)
   }
 
   public ProtobufDataCodec(SymbolTable symbolTable, boolean supportsASCIIOnlyStrings)
+  {
+    this(symbolTable, supportsASCIIOnlyStrings, false);
+  }
+
+  public ProtobufDataCodec(SymbolTable symbolTable,
+      boolean supportsASCIIOnlyStrings,
+      boolean useFixedLengthEncodingForFloatingPointValues)
```

Comment:
Great idea. I am going to create the config class as a separate top level class in the same package, so that it can be used by your streaming version in the future.